server/routes/ad.ts

```diff
@@ -963,11 +963,15 @@ export default class AdService {
 
       const detectorResult: AnomalyResult[] = [];
       const featureResult: { [key: string]: FeatureResult[] } = {};
+      
       get(response, 'hits.hits', []).forEach((result: any) => {
+        console.log("result: " + JSON.stringify(result))
         detectorResult.push({
           startTime: result._source.data_start_time,
           endTime: result._source.data_end_time,
           plotTime: result._source.data_end_time,
+          contributions: result._source.anomaly_grade > 0 
+            ? result._source.relevant_attribution : {},
```

Comment:
shouldn't the default value be an empty array?

Reply:
the type is defined as `contributions: { [key: string]: FeatureContributionData };`